flask_hw3/blueprints/supermarkets/supermarket.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from flask import Blueprint, render_template, request, session, url_for, redirect
+from werkzeug.utils import secure_filename
+
+from supermarket_db import SUPERMARKET_DB, SupermarketTemplate
+
+supermarket_page = Blueprint('supermarket_page', __name__,
+                             template_folder='./templates',
+                             static_folder='static',
+                             static_url_path='/supermarkets')
+
+
+@supermarket_page.route('/supermarket', methods=['GET', 'POST'])
```

Comment:
for different http method better use different routes, it removes if, else statements

Reply:
Well, yes) Even though you are absolutely right and it looks like crutch-oriented programming, it is said in homework that either route '/product' or '/supermarket' should receive 'get' and 'post' queries... So I did that.